packages/react-textarea/src/components/TextArea/useTextAreaStyles.ts

```diff
@@ -0,0 +1,273 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { TextAreaState } from './TextArea.types';
+import { tokens } from '@fluentui/react-theme';
+
+export const textAreaClassName = 'fui-TextArea';
```

Comment:
This export and `textAreaElementClassName` should be removed in favor of a [static classnames object](https://github.com/microsoft/fluentui/pull/21206).

Example from Button: https://github.com/microsoft/fluentui/pull/20977/files#diff-f6fb812b1e0ffe16b643e1d8f8db1240a2d67e6d7492df459a4597cac9b5aca5R7-R10